src/node/stage/approved.rs

```diff
@@ -1423,14 +1842,33 @@ impl Approved {
         Ok(())
     }
 
+    fn add_force_gossip_peer(
+        &mut self,
+        elders_info: &EldersInfo,
+        old_prefix: Prefix<XorName>,
+        was_elder: bool,
+    ) {
+        if was_elder
+            && (elders_info.prefix == old_prefix || elders_info.prefix.is_extension_of(&old_prefix))
+        {
+            for (_, id) in elders_info.elders.iter() {
```

Comment:
```rust
for id in elders_info.elders.values()
```
(I think clippy should be complaining about this)

Reply:
will do. however strangely clippy doesn't complain about this.